packages/core/src/utils/getRootSpan.ts

```diff
@@ -8,8 +8,8 @@ import type { Span } from '@sentry/types';
  *
  * If the given span has no root span or transaction, `undefined` is returned.
  */
-export function getRootSpan(span: Span): Span | undefined {
+export function getRootSpan(span: Span | undefined): Span | undefined {
   // TODO (v8): Remove this check and just return span
   // eslint-disable-next-line deprecation/deprecation
-  return span.transaction;
+  return span && span.transaction;
```

Comment:
needed this because the integration should only be changing the name of the root span (for transaction name paramaterization).

Reply:
hmm not sure, is it not cleaner to do `span ? getRootSpan(span) : undefined`, or do we need this so often? 🤔

Reply:
It means I can do `getRootSpan(getActiveSpan())`, which helps with DX a lot